src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/helpers/AdsComparison.java

```diff
@@ -49,13 +51,11 @@ private void changeOpacity(String selector, int value, WebDriver driver) {
     javascriptActions.changeElementOpacity(selector, value);
   }
 
-  public boolean compareImageWithScreenshot(final String pathToImage,
-                                            final WebElement element,
-                                            final WebDriver driver) {
+  public boolean compareImageWithScreenshot(final String pathToImage, final WebDriver driver) {
     BufferedImage expectedImage = imageEditor.fileToImage(new File(pathToImage));
-    BufferedImage actualImage = imageEditor.fileToImage(
-        shooter.captureWebElement(element, driver));
-    return imageComparison.areImagesTheSame(actualImage, expectedImage);
+    BufferedImage actualImage = imageEditor.fileToImage(new File("src/test/resources/adsResources/test3.png"));
+//        shooter.capturePage(driver));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)